.github/workflows/deb-packaging.yml

```
@@ -219,15 +219,15 @@ jobs:
             echo "dput exited with ${retval}"
             if ! grep -q "ssh: Could not resolve hostname package-drop.psonet: Name or service not known" ${tmpfile}; then
               rm ${tmpfile}
-              if ((retval != 0)); then
+              if [[ ${retval} != 0 ]]; then
                 exit ${retval}
               else
-                return
+                return 0
               fi
             fi
             if ((++i <= max_retries)); then
               sleep 10
-              echo "Retry ${i}/${max_retries}:"
+              echo "${COLOR_BLUE}Retry ${i}/${max_retries}:"
```

Do we need to reset the color after printing?

```suggestion
              echo "${COLOR_BLUE}Retry ${i}/${max_retries}:${COLOR_RESET}"
```

---

We don't do it elsewhere, so I guess the answer is no.